newdle/client/src/components/UserSearchResults.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Divider, Icon, List} from 'semantic-ui-react';
+
+import styles from './UserSearchResults.module.scss';
+
+export default function UserSearchResults({results: {users, total}, onAdd, isAdded}) {
+  return total !== 0 ? (
+    <>
+      <Divider horizontal>{`${total} users`}</Divider>
+      <List divided relaxed selection>
+        {users.map(user => (
+          <List.Item key={user.email} className={styles['result-item']}>
+            <List.Content floated="right">
+              {isAdded(user) ? (
+                <Icon name="checkmark" color="green" size="large" />
+              ) : (
+                <Icon name="plus" size="large" onClick={() => onAdd(user)} />
+              )}
+            </List.Content>
+            <List.Icon size="large" name="user" verticalAlign="middle" />
+            <List.Content>{user.name}</List.Content>
```

Comment:
What do you think about displaying user's email in the results as well? We allow searching by the email address so I think it would be good to display it. (I can also imagine a situation when we search by name and we get the same person twice because s/he is has registered twice with different email addresses. Then it would be also good to display the address to know which one we are choosing).